gramps/gui/viewmanager.py

```diff
@@ -107,148 +108,20 @@
 # Constants
 #
 #-------------------------------------------------------------------------
-if is_quartz():
-    try:
-        import gi
-        gi.require_version('GtkosxApplication', '1.0')
-        from gi.repository import GtkosxApplication as QuartzApp
-        _GTKOSXAPPLICATION = True
-    except:
-        print("Failed to import gtk_osxapplication")
-        _GTKOSXAPPLICATION = False
-else:
-    _GTKOSXAPPLICATION = False
+# if is_quartz():
```

Comment:
Just remove this?  Are there still Mac issues?

Reply:
I did quite a bit of testing on my OSX VM, I think we are in pretty good shape.  John Ralls also looked at it and seems to concur.  The Gtk.Application and related APIs have covered most of the bases for OSX that GtkosxApplication used to deal with, there were only a few items that had not been fully covered as of the Gtk version that we are packaging for MAC usage.  These items are taken care of with a few scattered is_quartz() calls.

There is one remaining mystery, as far as I am concerned.  OSX automagically adds a few menu items for things like insert emojii, or speech recognition.  For some reason I have not been able to determine, these are not working for this PR code.  On the bright side, even though they were added with our older code, they crashed Gramps if you tried to use them.  On the whole I think this is a win.  If I ever have too much time on my hands, I might try to set up a full MAC debug environment so I can get a better look inside their API code and try to resolve this.  Or perhaps John might give it a go...